### ISSUE TITLE
Java Error opening a password protected Excel workbook

### ISSUE THREAD

**Issue by SlavaKozlov** [miraisolutions/xlconnect#103]
For XLConnect after 0.2-15 having trouble opening an Excel workbook with a password.

Error

> Error: NoClassDefFoundError (Java): com/microsoft/schemas/office/x2006/encryption/EncryptionDocument$Factory

This is the same error as in  [issue #61](https://github.com/miraisolutions/xlconnect/issues/61).

To reproduce (same Test Excel file as in issue #61)

```
filepath <- file.path("~/Downloads", "TestWorkbook.xlsx")
wb <- loadWorkbook(filepath, create=F, password = "pass")
```

I tried a bit to search for the missing Java class in the POI Jar files in the XLConnectJars/java directory but with no luck. I notice that the most recent release says POI files are not included (?).

environment:

```
R version 3.6.3 (2020-02-29)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 17763)

Matrix products: default

locale:
[1] LC_COLLATE=English_United States.1252  LC_CTYPE=English_United States.1252    LC_MONETARY=English_United States.1252
[4] LC_NUMERIC=C                           LC_TIME=English_United States.1252    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base

other attached packages:
[1] XLConnectJars_1.0.0 XLConnect_1.0.1   

```

My workaround right now is to install the 0.2-15 version in a seperate directory and use that to read password protected Excel files.

Thank you

**Comment by spoltier** [comment]
Hi, as indicated in issue #102, there may have been some issue installing. It seems like no error is being produced in some cases even though the installation was not completed.
See [my comment on that issue](https://github.com/miraisolutions/xlconnect/issues/102#issuecomment-606014144) for a workaround.

**Comment by SlavaKozlov** [comment]
Thanks a lot for looking into this and noticing the similar issue.

I removed XLConnect and XLConnectJars and tried reinstalling just the XLConnect package. There were no install errors.

I continue to get the same error as above with the password protected file. 

I have the following jar files in the {lib}\XLConnect\java folder after install:
```
commons-collections4.jar
commons-compress.jar
commons-math3.jar
curvesapi-1.06.jar
ooxml-schemas.jar
poi-ooxml.jar
poi.jar
XLConnect-1.0.0.jar
xmlbeans.jar
```

Since my initial post, I have upgraded to ```R version 4.0.0 (2020-04-24)``` but I had the same issue in 3.6.3.

More attempts that also proved fruitless:

* I tried building from source for 1.0.1 branch. 
```devtools::install_github("miraisolutions/xlconnect", ref = "1.0.1")```

* I tried building from master:
```devtools::install_github("miraisolutions/xlconnect")```

These are the jar files downloaded during both builds:
```
trying URL 'https://repo1.maven.org/maven2/org/apache/poi/poi-ooxml/4.1.1/poi-ooxml-4.1.1.jar'
Content type 'application/java-archive' length 1859884 bytes (1.8 MB)
==================================================
downloaded 1.8 MB

trying URL 'https://repo1.maven.org/maven2/org/apache/poi/poi/4.1.1/poi-4.1.1.jar'
Content type 'application/java-archive' length 2827712 bytes (2.7 MB)
==================================================
downloaded 2.7 MB

trying URL 'https://repo1.maven.org/maven2/org/apache/commons/commons-compress/1.19/commons-compress-1.19.jar'
Content type 'application/java-archive' length 615064 bytes (600 KB)
==================================================
downloaded 600 KB

trying URL 'https://repo1.maven.org/maven2/org/apache/xmlbeans/xmlbeans/3.1.0/xmlbeans-3.1.0.jar'
Content type 'application/java-archive' length 2579738 bytes (2.5 MB)
==================================================
downloaded 2.5 MB

trying URL 'https://repo1.maven.org/maven2/org/apache/commons/commons-collections4/4.4/commons-collections4-4.4.jar'
Content type 'application/java-archive' length 751914 bytes (734 KB)
==================================================
downloaded 734 KB

trying URL 'https://repo1.maven.org/maven2/org/apache/commons/commons-math3/3.6.1/commons-math3-3.6.1.jar'
Content type 'application/java-archive' length 2213560 bytes (2.1 MB)
==================================================
downloaded 2.1 MB

trying URL 'https://repo1.maven.org/maven2/org/apache/poi/ooxml-schemas/1.4/ooxml-schemas-1.4.jar'
Content type 'application/java-archive' length 16356588 bytes (15.6 MB)
==================================================
downloaded 15.6 MB
```

**Comment by spoltier** [comment]
Closed as part of #117, included in release 1.0.2
